src/components/Preview/Preview.styled.ts

```diff
@@ -4,44 +4,72 @@ import styled from 'styled-components'
 // Colors
 import { colors } from '../../theme/colors'
 
-const { white, grandis, easternBlue, greenHaze, slateGray, mineShaft, iron, outerSpace } = colors
+const {
+  white,
+  grandis,
+  easternBlue,
+  greenHaze,
+  slateGray,
+  mineShaft,
+  iron,
+  outerSpace,
+  mako
+} = colors
 
 export const StyledPreview = styled.div`
   background-color: ${white};
   border: 1px solid ${mineShaft};
+  display: grid;
+  grid-template-columns: 250px 1fr;
+  /* flex-direction: column;
+  justify-content: space-between; */
+  width: 100%;
+  height: 100vh;
+
+  h1 {
+    border-bottom: 1px solid #252525;
+    padding-left: 1rem;
+    font-size: 1.5rem;
+    line-height: 36px;
+    width: 100%;
+  }
+`
+export const StyledMenu = styled.div`
+  border-right: 1px solid #252525;
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
-  width: 1024px;
-  margin: 0 auto;
-  position: relative;
+  max-height: 100vh;
+  overflow: auto;
+`
+export const StyledMenuItem = styled.a`
+  padding: 1rem;
+  border-bottom: 1px solid #eaeaea;
```

Comment:
Add this color to the colors object on /theme/colors.ts, you can find the color name here: https://chir.ag/projects/name-that-color/#EAEAEA